packages/plugin-rag/src/index.ts

```
@@ -0,0 +1,502 @@
+            // Handled by the readyPromise setup above for initial readiness
+            break;
+          case 'KNOWLEDGE_ADDED':
+            logger.info(
+              `RagService: Worker confirmed knowledge added for doc ${message.payload.documentId}, count: ${message.payload.count} (agent: ${message.payload.agentId}).`
+            );
+            break;
+          case 'PDF_MAIN_DOCUMENT_STORED':
+            logger.debug(
+              `RagService: Worker confirmed PDF main document stored for clientDocId ${message.payload.clientDocumentId}, dbDocId: ${message.payload.storedDocumentMemoryId} (agent: ${message.payload.agentId}).`
+            );
+            const callback = this.pendingPdfCallbacks.get(message.payload.clientDocumentId as UUID);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)